pallets/xcm-app/src/lib.rs

```diff
@@ -144,6 +154,15 @@ pub mod pallet {
     pub type MultilocationToAssetId<T: Config> =
         StorageMap<_, Blake2_256, MultiLocation, AssetId, OptionQuery>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn message_trap)]
+    pub type BridgeAssetTrap<T: Config> =
```

Comment:
It is better to leave a comment for the storage. Like, `A storage for tokens that failed during refund`.